py/client/tests/test_console.py

```diff
@@ -49,6 +49,9 @@ def vectorized_func(x, y):
         pa_table = self.session.open_table('demo_table').to_arrow()
         df = pa_table.to_pandas()
         self.assertEquals(1000, len(df.index))
+        pa_table = self.session.ticket_to_table('s/demo_table'.encode(encoding='ascii')).to_arrow()
```

Comment:
this is "wrong" - the ticket_to_table method says that it takes a string, but you're passing a `bytes`

```python
python
Python 3.11.3 (main, Jun  5 2023, 09:32:32) [GCC 13.1.1 20230429] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> type('s/demo_table'.encode(encoding='ascii'))
<class 'bytes'>
```